.trivyignore

```diff
@@ -1,6 +1,14 @@
 # Jenkins CVEs
-CVE-2023-20863
 CVE-2016-1000027
+CVE-2023-20863
```

Comment:
Is not possible to address these issues?

Reply:
The issues are with things packaged in the jenkins.war file which is already the latest version. Perhaps these issues will get resolved in later patches!